torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -163,7 +165,7 @@ def idxs_to_tokens(self, idxs: torch.LongTensor) -> List:
 def kenlm_lexicon_decoder(
```

Comment:
Maybe rename the `kenlm_lexicon_decoder` to `lexicon_decoder` to cover the empty lm case, what do you think?

Reply:
I think that makes sense.

Also I think that the namespace can be `torchaudio.prototype.decoder`. (not the scope of this PR though)